app/src/androidTest/java/com/github/sdp_begreen/begreen/models/TrashCategoryTest.kt

```diff
@@ -0,0 +1,104 @@
+package com.github.sdp_begreen.begreen.models
+
+import android.os.Parcel
+import com.github.sdp_begreen.begreen.matchers.ContainsPropertyMatcher
+import kotlinx.coroutines.runBlocking
+import org.hamcrest.CoreMatchers
+import org.hamcrest.MatcherAssert
+import org.junit.Assert
+import org.junit.Test
+import com.github.sdp_begreen.begreen.R
+import com.google.android.gms.maps.model.BitmapDescriptorFactory
+
+class TrashCategoryTest {
```

Comment:
It looks to me as though this test class could be in the `test `directory and not the `androidTest `one.